app/src/main/java/com/github/kpdandroid/ui/KeypointPainter.kt

```diff
@@ -0,0 +1,45 @@
+package com.github.kpdandroid.ui
+
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Canvas
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.graphics.Paint
+import androidx.compose.ui.graphics.PointMode
+import androidx.compose.ui.graphics.asAndroidBitmap
+import androidx.compose.ui.graphics.nativeCanvas
+import android.graphics.Canvas as NativeCanvas
+import android.graphics.Color as NativeColor
+
+private const val KEYPOINT_WIDTH = 10f
+
+class KeypointPainter(image: ImageBitmap? = null, pointColor: Color = Color.Blue) {
+    private var bitmap = image?.asAndroidBitmap()
+    private val canvas = image?.let { Canvas(it) } ?: Canvas(NativeCanvas())
```

Comment:
why do you use compose canvas instead native one?

Reply:
Native canvas' `drawPoints` method is much less convenient to use: it accepts an array of floats which represent 2D coordinates but aren't  even paired